diff --git a/drivers/kernelsu/Kbuild b/drivers/kernelsu/Kbuild
index 5b45090..e4b1537 100644
--- a/drivers/kernelsu/Kbuild
+++ b/drivers/kernelsu/Kbuild
@@ -28,6 +28,7 @@ kernelsu-objs += policy/feature.o
 
 kernelsu-objs += runtime/boot_event.o
 kernelsu-objs += runtime/ksud_integration.o
+kernelsu-objs += runtime/properties.o
 
 kernelsu-objs += selinux/selinux.o
 kernelsu-objs += selinux/rules.o
